products/eclipse-jetty.md

```diff
@@ -0,0 +1,120 @@
+title: Eclipse Jetty
+category: server-app
+tags: eclipse web-server
+iconSlug: eclipsejetty
+permalink: /eclipse-jetty
+releasePolicyLink: https://eclipse.dev/jetty/download.php
+changelogTemplate: https://github.com/jetty/jetty.project/releases/tag/jetty-__LATEST__
+releaseDateColumn: true
+eolColumn: "Community Support"
+extendedSupportColumn: "Extended Support"
+
+identifiers:
+-   purl: pkg:maven/org.eclipse.jetty/jetty-server
+
+auto:
+-   maven: org.eclipse.jetty/jetty-server  
+
+# eol status and minJavaVersion can be found on https://eclipse.dev/jetty/download.php
+releases:
+-   releaseCycle: "12"
+    releaseDate: 2023-08-07
+    eol: false
+    extendedSupport: true
+    minJavaVersion: 17
+    latest: "12.0.3"
+    latestReleaseDate: 2023-10-30
+
+# https://github.com/jetty/jetty.project/issues/10485
+-   releaseCycle: "11"
+    releaseDate: 2020-12-07
+    eol: 2024-01-01
+    extendedSupport: true
+    minJavaVersion: 11
+    latest: "11.0.18"
+    latestReleaseDate: 2023-10-30
+
+# https://github.com/jetty/jetty.project/issues/10485
+-   releaseCycle: "10"
+    releaseDate: 2020-12-07
+    eol: 2024-01-01
+    extendedSupport: true
+    minJavaVersion: 11
+    latest: "10.0.18"
+    latestReleaseDate: 2023-10-30
+
+# https://github.com/jetty/jetty.project/issues/7958
+-   releaseCycle: "9.4"
+    releaseDate: 2016-12-07
+    eol: 2022-06-01
+    extendedSupport: true
```

Comment:
Are all versions released after the end of community support made available freely to everybody ? If yes I think this must be documented in the description.

Reply:
That's not clear to me either Marc. From what I understand on these pages (https://github.com/jetty/jetty.project/issues/7958) the security releases are still available to the public. I'm unsure if the releases which are triggered by webtide customers are publically available.

Reply:
I figured it out, support is divided into two phases:

- community support : bug and security fixes,
- security support: critical bug and security fixes.

When a release is in the security support phase, extended support is still available if needed, and looks like some changes are backported to publicly available version, such as https://github.com/jetty/jetty.project/releases/tag/jetty-9.3.30.v20211001.

I updated the page accordingly, and did not detailed too much the commercial support as I still don't know if all sponsored changes are made available to everybody or not.